src/collections/Hero.tsx

```diff
@@ -103,17 +110,24 @@ export const Hero = ({
         theme ? theme : 'dark',
         className
       )}
-      style={{backgroundColor: bgColor, backgroundImage: `url(${bgImageOutput})`}}
+      style={{backgroundColor: bgColor}}
       aria-labelledby={HeroHeadingId}
     >
+      <img
+        alt=""
```

Comment:
Do we need to add aria stuff to let the browser know that this is a background and not foreground main content?
Also, what does an empty `alt` tag achieve?

Reply:
I'm on it

Reply:
I updated alt and caption as it was not working everywhere